packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -1993,6 +1993,9 @@ function commitRootImpl(root, renderPriorityLevel) {
     while (nextEffect !== null) {
       const nextNextEffect = nextEffect.nextEffect;
       nextEffect.nextEffect = null;
+      if (nextEffect.effectTag & Deletion) {
+        nextEffect.sibling = null;
```

Comment:
Seems a little unfortunate, since all other Fiber fields get cleared during mutation by `detachFiber`.

I wonder if we should just move the call to `detachFiber` here instead, and add `sibling = null` to it? That way we'd keep all of the nulling out of fields in the same place.

Reply:
I originally did that, but then we had a bunch of code that expects `return` to be `null` before now, so then I ended up having to add `fiber.return = null` in the commit phase and I just thought it seemed not that different (given we want to really detatch there, we're blocked because of `findDOMNode`).

Reply:
D'oh

Maybe we could have two methods, e.g. `detachFiberMutation` and `detachFiberAfterEffects` (or something, I dunno, names are hard), so it's a little more explicit in the source than a buried `.sibling = null` ?

Reply:
I've made the renames to what you mentioned, seemed logical.